app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -122,6 +144,24 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         return currView;
     }
 
+    private void fetchImage(String id, ImageCallback callback){
+        AsyncHttpClient client = new AsyncHttpClient();
+        String getUrl = SHOW_DETAIL_URL + id + ADD_API_KEY + getContext().getString(R.string.movie_api_key);
+
+            // call URL and parse through JSON
+            client.get(getUrl, new JsonHttpResponseHandler() {
+                @Override
+                public void onSuccess(int i, Headers headers, JSON json) {
+                    callback.onSuccess(json);
+                }
+
+                @Override
+                public void onFailure(int i, Headers headers, String s, Throwable throwable) {
+                    callback.onFailure(i);
+                }
+            });
+    }
```

Comment:
Do you think we could pull this out into a different class?  And then, we can just reuse this same call on the movies detail screen right?

Reply:
I have no more API call in the details page actually! Although, if the image on the FeedFragment does not load by the time I navigate to the page then there is no image loaded